protected/humhub/modules/content/widgets/StateBadge.php

```diff
@@ -18,23 +24,34 @@ class StateBadge extends Widget
 {
     public ?ContentActiveRecord $model;
 
+    /**
+     * @throws \yii\base\InvalidConfigException
+     * @throws \Exception
+     */
     public function run()
     {
         if ($this->model === null) {
             return '';
         }
 
-        if ($this->model->content->state === Content::STATE_DRAFT) {
-            return Html::tag(
-                'span', Yii::t('ContentModule.base', 'Draft'),
-                ['class' => 'label label-danger label-state-draft']
-            );
-        } elseif ($this->model->content->state === Content::STATE_DELETED) {
-            return Html::tag(
-                'span', Yii::t('ContentModule.base', 'Deleted'),
-                ['class' => 'label label-danger label-state-deleted']
-            );
+        switch ($this->model->content->state) {
+            case Content::STATE_DRAFT:
+                return Html::tag('span', Yii::t('ContentModule.base', 'Draft'),
+                    ['class' => 'label label-danger label-state-draft']
+                );
+            case Content::STATE_SCHEDULED:
+                $scheduledDateTime = new DateTime($this->model->content->scheduled_at, new DateTimeZone('UTC'));
+                return Html::tag('span', Yii::t('ContentModule.modules', 'Scheduled at {dateTime}', [
```

Comment:
We should also sue `ContentModule.base` as message category here.

Reply:
@luke- Fixed in the commit https://github.com/humhub/humhub/pull/6116/commits/87b919b497c35ebe54d17175820441db0a3e8323